README.md

```diff
@@ -284,47 +282,48 @@ public class FailoverSampleApp2 {
->2. It is highly recommended that you use the cluster and read-only cluster endpoints instead of the direct instance endpoints of your Aurora cluster, unless you are confident about your application's usage of instance endpoints. Although the Driver will correctly failover to the new writer instance when using instance endpoints, usage of these endpoints are discouraged because individual instances can spontaneously change reader/writer status when failover occurs. The driver will always connect directly to the instance specified if an instance endpoint is provided, so a write-safe connection cannot be assumed if the application uses instance endpoints.
+>2. It is highly recommended that you use the cluster and read-only cluster endpoints instead of the direct instance endpoints of your Aurora cluster, unless you are confident about your application's usage of instance endpoints. Although the Driver will correctly failover to the new writer instance when using instance endpoints, use of these endpoints is discouraged because individual instances can spontaneously change reader/writer status when failover occurs. The driver will always connect directly to the instance specified if an instance endpoint is provided, so a write-safe connection cannot be assumed if the application uses instance endpoints.
 
 ### Connection Plugin Manager
-Connection plugin manager initializes, triggers, and cleans up a chain of connection plugins. Connection plugins are widgets attached to each `Connection` objects to help execute additional or supplementary logic related to that `Connection`. [Enhanced Failure Monitoring](https://github.com/awslabs/aws-mysql-jdbc#enhanced-failure-monitoring) is one example of a connection plugin. 
+The connection plugin manager initializes, triggers, and cleans up a chain of connection plugins. Connection plugins are widgets attached to each `Connection` object to help execute additional or supplementary logic related to that `Connection`. [Enhanced Failure Monitoring](https://github.com/awslabs/aws-mysql-jdbc#enhanced-failure-monitoring) is one example of a connection plugin. 
```

Comment:
The more I look at it the more awkward this line looks. 
> Enhanced Failure Monitoring is one example of a connection plugin.

I think we should refer to the feature as Enhanced Failure Monitoring and refer the plugin as Enhanced Failure Monitoring Connection Plugin

Reply:
Would it be better phrased:

Enhanced Failure Monitoring is just one example of a connection plugin; you can use the 'Enhanced Failure Monitoring' Connection plugin to implement the functionality.

Reply:
I propose we rename `Enhanced Failure Monitoring` to `Enhanced Failure Monitoring Plugin`.

Reply:
Great, let's do that name change. I'll merge this PR in and we can open up another PR that does the renaming.